lib/utf8.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+module.exports.decode = function (input) {
+  if (Buffer.isBuffer(input) === false) {
```

Comment:
But the issue is that the function as implemented is not a decoder. Decoding literally means going from bytes to strings. Maybe what you want is to create a separate function, encodeThenDecode, and add a long comment explaining that this is to work around the limitations of subpar packages.